Makefile

```diff
@@ -17,6 +17,7 @@ protobufs:
 	make -C web
 
 models:
+	rm -f models/*.go
```

Comment:
AFAIK cleanup should go in it's own rule so something like:
```Makefile
models: clean
    PGSSLMODE=disable xo pgsql://soapbox@localhost:54320/soapbox_dev -o models --template-path models/templates/

.PHONY: models, clean
clean:
    rm -f models/*.go
```
does that make sense?

Reply:
I'm a little wary of making that its own `clean` rule, just because it's not really something that should ever be run on its own. Those files are versioned, but I noticed that when I removed a table before `xo` didn't understand that it should remove that file. This just makes it so that `make models` will delete all the existing models files and then regenerate them, such that deleting a table from the schema will delete the code from the generator.

Also, it seems like all of these targets should be marked `PHONY`, but maybe that's for another day.